common/src/main/java/com/linecorp/centraldogma/internal/api/v1/PushResultDto.java

```diff
@@ -34,9 +35,15 @@ public class PushResultDto {
     private final Revision revision;
     private final String pushedAt;
 
-    public PushResultDto(Revision revision, long commitTimeMillis) {
+    @JsonCreator
+    public PushResultDto(@JsonProperty("revision") Revision revision,
+                         @JsonProperty("pushedAt") Instant pushedAt) {
         this.revision = requireNonNull(revision, "revision");
-        pushedAt = ISO_INSTANT.format(Instant.ofEpochMilli(commitTimeMillis));
+        this.pushedAt = ISO_INSTANT.format(pushedAt);
+    }
+
+    public PushResultDto(Revision revision, long commitTimeMillis) {
```

Comment:
nit; for consistent naming

```suggestion
    public PushResultDto(Revision revision, long pushedAt) {
```